prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -492,7 +492,7 @@ has_detectors_reco MFT MCH && has_detector_matching MFTMCH && add_W o2-globalfwd
 
 # ---------------------------------------------------------------------------------------------------------------------
 # Writers for detectors whose reco workflow cannot write the output
-has_detector_reco TRD && ( [[ -z "$DISABLE_ROOT_OUTPUT" ]] || needs_root_output o2-trd-digittracklet-writer ) && ! has_detector_from_global_reader TRD && add_W o2-trd-digittracklet-writer
+[[ $CTFINPUT == 1 ]] && has_detector_reco TRD && ( [[ -z "$DISABLE_ROOT_OUTPUT" ]] || needs_root_output o2-trd-digittracklet-writer ) && ! has_detector_from_global_reader TRD && add_W o2-trd-digittracklet-writer
```

Comment:
I don't understand the CTFINPUT here?
Why can't we write TRD tracklets if we process raw data?

Reply:
We can and this is then done automatically by the o2-trd-datareader. When starting from CTF we don't have the raw parser and no device would write the TRD data otherwise